docs/src/main/asciidoc/_developer-guide/JPA_Reference_Guide.adoc

```diff
@@ -705,16 +650,8 @@ Manifest-Version: 1.0
 Dependencies: org.hibernate
 ....
 
-If you use the Hibernate native api in your application and also use the
-JPA api to access the same entities (from the same Hibernate
-session/EntityManager), you could get surprising results (e.g.
-HibernateSession.saveOrUpdate(entity) is different than
-EntityManager.merge(entity). Each entity should be managed by either
-Hibernate native API or JPA code.
-
 [[injection-of-hibernate-session-and-sessionfactoryinjection-of-hibernate-session-and-sessionfactory]]
-== Injection of Hibernate Session and SessionFactoryInjection of
-Hibernate Session and SessionFactory
+== Injection of Hibernate Session and SessionFactoryInjection of Hibernate Session and SessionFactory
```

Comment:
This titles looks weird, as if it were repeated.

Reply:
fixed, thanks!